FWCore/ParameterSet/python/TreeCrawler.py

```diff
@@ -120,7 +120,8 @@ def import_module(self,partnam,fqname,parent):
         if r is not None:
             self._depgraph.setdefault(self._last_caller.__name__,{})[r.__name__] = 1
         return r
-    def load_module(self, fqname, fp, pathname, (suffix, mode, type)):
+    def load_module(self, fqname, fp, pathname, xxx_todo_changeme):
```

Comment:
That actually comes straight from the futurize conversion, but if you have a suggestion (I think this is code you own) I will be happy to update the branch.